website/src/data/users.js

```diff
@@ -267,6 +267,14 @@ const users = [
     source: 'https://github.com/FoalTS/foal/tree/master/docs',
     tags: ['highlight', 'design', 'large', 'versioning', 'i18n'],
   },
+  {
+    title: 'Gladys Assistant',
+    description: 'A privacy-first, open-source home assistant',
+    preview: require('./showcase/gladys-assistant.png'),
+    website: 'https://gladysassistant.com/',
+    source: 'https://github.com/GladysAssistant/v4-website',
+    tags: ['design', 'large'],
```

Comment:
```suggestion
    tags: ['i18n'],
```

Just going to change the tags here, as your site has i18n, but it is not particularly very large or have a very customized design